tests/compatibility-test.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python
+import logging
+import os
+import sys
+import tempfile
+import time
+import unittest
+from string import Template
+
+import docker
+
+ray_version = '1.8.0'
+ray_image = "rayproject/ray:1.8.0"
+
+kindcluster_config_file = 'tests/config/cluster-config.yaml'
+
+raycluster_spec_template = 'tests/config/ray-cluster.mini.yaml.template'
+raycluster_service_file = 'tests/config/raycluster-service.yaml'
+
+logger = logging.getLogger(__name__)
+
+
+def shell_run(cmd):
+    logger.info(cmd)
+    return os.system(cmd)
+
+
+def shell_assert_success(cmd):
+    assert shell_run(cmd) == 0
+
+
+def shell_assert_failure(cmd):
+    assert shell_run(cmd) != 0
+
+
+def create_cluster():
+    shell_assert_success(
+        'kind create cluster --config {}'.format(kindcluster_config_file))
+
+
+def apply_kuberay_resources():
+    shell_assert_success(
+        'kubectl apply -k github.com/ray-project/kuberay/manifests/cluster-scope-resources')
```

Comment:
Did we update the image `kubectl kustomize edit set image kuberay/operator=kuberay/operator:${commit_hash}` here to test against nightly image?